src/pages/ServiceDetails/ServiceDetailsPage.tsx

```diff
@@ -145,24 +145,6 @@ class ServiceDetails extends React.Component<ServiceDetailsProps, ServiceDetails
     return {};
   }
 
-  searchValidation(parsedSearch: ParsedSearch) {
```

Comment:
I don't know why, it smells bad to me. Perhaps in the refactor was left unused but it is somewhat necessary.
Good test scenarios coverage would be useful. I would like to dig a little bit more on that and perhaps not merged right before today's release. Thoughts?

Reply:
Well, I think if i'm having the bug, other could have to. I don't know why it happens to me. Not using maistra? Not having cluster rights? I see on the backend some differentiations related to that.

Reply:
@xeviknal about the `searchValidation` being removed: that's unused anyway, I'm not removing any call to it. So that's going to be in the release.

Reply:
A quick search confirms this is unused.  And uncommented, so +1 to removal.